### DISCUSSION TITLE
Help me please

### DISCUSSION THREAD

**Discussion by jackhallloween21** [sandeepyadav1478/Transfer-files-from-google-to-terabox-cloud#3] [Q&A]
l have created a discussion

**Comment by jackhallloween21** [comment]
![snip](https://user-images.githubusercontent.com/78194804/125399270-f994a000-e3cd-11eb-8b10-5ea99a2f9d1f.PNG)

**Reply by sandeepyadav1478** [reply]
2nd node of code creates this file.

**Reply by sandeepyadav1478** [reply]
have you successfully executed 2nd block?

**Reply by jackhallloween21** [reply]
previous nodes ran properly 
![image](https://user-images.githubusercontent.com/78194804/125399450-40829580-e3ce-11eb-99c0-79c5f5dca330.png)
another error it  was showing
![image](https://user-images.githubusercontent.com/78194804/125399533-5b550a00-e3ce-11eb-8931-5fb1ae4f1d7a.png)

**Reply by sandeepyadav1478** [reply]
k, so this code taking file from google drive,
U have to change 1st parameter to the file location. where you downloaded it.

**Reply by jackhallloween21** [reply]
i didn't get it. I got the download link from google take out

**Reply by sandeepyadav1478** [reply]
yeah then execute 2nd block after pasting link in it.

**Reply by sandeepyadav1478** [reply]
it will download that file on colab directory

**Reply by jackhallloween21** [reply]
i pasted my link here
# downloading anytype of file from server
import urllib.request
# following variable keep target file location with file extension
# It is google photos export download link
tar_loc="10.zip"
urllib.request.urlretrieve('https://00f74ba44b80139f654a7d45a18cb302771c1e93dd-apidata.googleusercontent.com/download/storage/v1/b/dataliberation/o/20210711T043243Z%2F-4326093206984207447%2F6048f8ff-aeeb-47ca-9321-3ec275e7f8a2%2F1%2F047d0489-e3b5-4a91-9008-92e2a0d0fc6a?jk=AFshE3UHL0I4XcxGYR4DZbUc5IDHAeGJ9bcY30ShgQJK_RZte7rR-SINkUXDnJhDjj0LEYvUGkF5yzv25fvop33hKALSZ0l5jrvh1E_kdIwTbzcahC15vPiP-uix4MHw5UmPTrrZG9aPRtv9X0m_ZewNKv8XX43oSlmxHOwNXY5jov4kmF5tvoY5Dl39pDbAY3S45WuBthEfu5dCCeocrVDGWVOSuZ0KMvCYPcsK-L5ep_Cg06K3nLOyNXclOLFrnNkhi0LfKiIOXtnL-8l_tQs9Bzz8p3Aaz4Q0EStTezipl24PBMEhlB9w_gjGvCU_g_rbgWJgfCXzzVjw1Yk76b6Oo9CCkCWJA8itTu9KC23O2UjSbdgci8c-k9YZTVq8qbZl6Rn7V0MxMlgYl7YD53RojVcf_F7O27xfdKVNXgXz4JJtvek0qKCJnna4HlcY5b7XHhGtmEIfBohjKkcuuIWy5jWT1Sv-HN-UYGIhQMBpA36Z2jhrUHwdKrOKk4gybjNe1v-yfv-_5-9RA4vOSsZHo94nQR0CMNvp3IZLSPeiaUHBXo3ZNXID1LZ4coRZLnjXLjewhkBaEJwmqoctxJZ-xGhJdQ1FSk_KpVvIMm-HaK7ab_t_LIbNcL_fM_D7Q0Rt-Jru-Hhsg0iJU4tCEMfMAPt5lCWysrpMTzHBA1i-oDWKVhO4QnB00EaV_jU3Dlie97cTLJ0IKbbaCkELbjtT5iWaAU5mvQi0Q1j1I4z-X-HvFBcxQVdJ53UCb4YVOnypa9jXmvUiZrSOx9MR13JQSobhK9kLGVUb5qbZdi_svT-g-7usMuCn9ZR6dJLPBrJQJo37Jac4O4xh0-4ncEAT2hZflVmD-Vaq_bwgVY1NyAqqH2qYshec2-1yO5gLZcIb7EZTSgl3unKsL12UVj3p53Kwnbcr1i-RursOI2FGIEhYrb8BjxJQcNFU3Do1y3TE5ZyYjO9K2XtdIt6XBFShUL_cSzQnAOevkBz7YPScs39-6LSC6ASpFY0-l40LYbUAwsQv9cdwKpFLhrxXSe5rrv4t_6_mpYAz6i_2NhTO86qdt8ogUH8jqsIGYCVGbOgMxvOlexjJJpLAtQqmXPhD&isca=1',tar_loc)

**Reply by sandeepyadav1478** [reply]
xd

**Reply by sandeepyadav1478** [reply]
this address!!!!

**Reply by sandeepyadav1478** [reply]
it is expired link.

**Reply by sandeepyadav1478** [reply]
you have to create new link.

**Reply by jackhallloween21** [reply]
i will insert this link in place of this
 file_path="/content/drive/MyDrive/gp_310521/10.zip"
?

**Reply by jackhallloween21** [reply]
> you have to create new link.

ok

**Reply by sandeepyadav1478** [reply]
no its target position, where you will put this file.

**Reply by sandeepyadav1478** [reply]
do you have google duo on ur phone?

**Reply by sandeepyadav1478** [reply]
coz i can help asap i can with screen sharing.

**Reply by jackhallloween21** [reply]
yes i have duo but meet has better screen sharing

**Reply by sandeepyadav1478** [reply]
meet?

**Reply by jackhallloween21** [reply]
i got the new link
https://00f74ba44bc21aabfd9db1195f8ba23dbd19a175ae-apidata.googleusercontent.com/download/storage/v1/b/dataliberation/o/20210711T043243Z%2F-4326093206984207447%2F6048f8ff-aeeb-47ca-9321-3ec275e7f8a2%2F1%2F047d0489-e3b5-4a91-9008-92e2a0d0fc6a?jk=AFshE3XMT5P8pra3bRj7LEBBRZUw0M0pZxJU7uW4uWbyHhWaDhuGFovBiAmahRTOrrY3PJ6EHJoPolMQV0Gsxtz4ksVFbGU2dW3DmUyptPDyo3rb7gt6VjPhtXJpbGPU9DmmfreCmtVZyohGqG2bI4S2xszDRqC5ue2GetphqCB8hn9HCQScaK-29lKU1JzrcDPE-JN7NdecmpeQoINI3ZmYYwCcjgWMrqiPilRqwDuOWLkNMtJgKQQjT-g5bALt2VYWtZJLbbFf4kH2Nn123UFwrIQN8yP1Nbav8w-SuIQTgF1FC7au5kj0VndxSrSY1cR2MpzDLpbN90Aqzjmhupy3Sf29Oqy-BpvsqjFEeHOXQsV5fAr6Hw3w6mJVC2I84fB5urIXaWIma0DNkN-M6N-Ulz04_Elr0QKM7_OF8AzFunTlaqgfmu8L0ZDGTx2BFa6d0mEZtDCRNHocyMNT5HFHQ1uk6_MJIjqVWf4Aai7N8Ebd_EAGOdZNulGOZAUqUKtq15byuSXlg_NmVBXB4cvjO6QZI63iUuGEU0JbAb2x6hgL9MV6ipNi7b95twLHjjFP56_29Lj5An6bzz4ZUS2LlhgMbOa-HpivRRRFjyjdmVhsdLBzG7ivUKurQtXX0EZHGGxXHESCPMVuAaNfsClonBxr42_DwoU7kq9Dvgc4mVI0xinvF6LWKmicF_iQ4DnTTLZxwc5y2x_31TrP9dtzAT5mjq2NitV96RkBN4SAUswQmDmYhtQTS_ZV8X7tcZSwRhPbuBKYIf2z1ZGRAFWIUbrGpNJYixBy4J7vw4Q5mRAyna_A61Ns4_kBmZkUZched5ZQZklABXJmsS0PPr9SyHEM7y_jSgCUhfBc0cQUUko8GqBmtqDV4Aj7Hz-wqLrNMMna3boyegZD5HEVH1kLtd9FMWAW36TpiqF6M4t95IfF-hBEmrVk8fI_Lk3U-NWpL8J8wOwYPmFZGkrnlpKihAWEQnJDz5zdGytEC8lJPqDIjP4iFicjVY9iC9I4CFRKO0Af3_JmPqJdQHueC-_m0mb_GB4Kn9KdhUcDvzIq7AH9SQjWyqLvXbTERZl9Yf_H8uIquxAPflWyvuXvAubp&isca=1

**Reply by jackhallloween21** [reply]
google meet

**Reply by sandeepyadav1478** [reply]
yeah its good one

**Reply by sandeepyadav1478** [reply]
use this link in 2nd block

**Reply by jackhallloween21** [reply]
ok i am sending the meet link

**Reply by jackhallloween21** [reply]
To join the meeting on Google Meet, click this link: meet.google.com/aik-iuum-qmw

Or open Meet and enter this code: aik-iuum-qmw

**Comment by jackhallloween21** [comment]
# downloading anytype of file from server
import urllib.request
# following variable keep target file location with file extension
# It is google photos export download link
tar_loc="10.zip"

this one ran properly took a long time and disk space started to decrease getting filled up

**Comment by jackhallloween21** [comment]
![image](https://user-images.githubusercontent.com/78194804/125399785-b0911b80-e3ce-11eb-982c-b2d6e0e8b5ab.png)
this one also ran properly i tried to run it again so it is telling file already exists

**Reply by sandeepyadav1478** [reply]
it means this folder already exists in your colab system.
